build.gradle

```diff
@@ -177,17 +177,18 @@ task runTestsWithTag(type: Test) {
         String terraServer = project.hasProperty("server") ? project.findProperty("server") : "broad-dev"
         environment "TERRA_SERVER", terraServer
 
+        // set which file in src/test/resources/testconfigs to use
+        // TODO(PR-1289): After tests running against Verily deployment are using Verily testconfig and not Broad,
```

Comment:
nit: PF-1289, I think you fixed this in the base branch so this change might be out of date

Reply:
Done